src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,12 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        Lottery ball2 = new Lottery();
+        Lottery ball3 = new Lottery();
```

Comment:
According to the checklist, you should use a loop for creating several objects of the same class. Instead of creating each `Lottery` instance separately, consider using a loop to create and store them in an array or list.